### DISCUSSION TITLE
Support of Alt Tab

### DISCUSSION THREAD

**Discussion by liumingedwin** [CuarzoSoftware/Louvre#78] [Ideas]

**Comment by Fox2Code** [comment]
@liumingedwin Louvre is a compositor, not a compositor library.

Louvre Views is an example compositor for Louvre, so I'll assume you mean you want Alt Tab support in Louvre Views.

**Comment by ehopperdietzel** [comment]
> @liumingedwin Louvre is a compositor, not a compositor library.

It’s the other way around.
